Terraform/SetupTerraformBackend/main.tf

```diff
@@ -0,0 +1,43 @@
+# Establish and configure Project 1 Terraform Locking State in S3
+
+
+# configure provider with proper credentials 
+provider "aws" {
+  version = "~> 2.33"
+  region = "us-west-2"
+  shared_credentials_file = "/Users/kindra/.aws/credentials"
```

Comment:
Kindra I think this line should to read: 

shared_credentials_file = "$HOME/.aws/credentials"

We need this to be dynamic based on the user correct? We can assume that $HOME will be /Users/User_Name